includes/Hooks.php

```diff
@@ -27,7 +27,10 @@ public function __construct( Config $config, CreateWikiHookRunner $hookRunner )
 		$this->config = $config;
 		$this->hookRunner = $hookRunner;
 	}
-
+	public static function onLoginFormValidErrorMessages( &$messages ) {
```

Comment:
This needs to:

1) Implement interface
2) Have no return value (remove `return true;`)
3) Fix param type declaration for `&$messages` to be `array &$messages`
4) Add `/** @inheritDoc */` comments
5) Use hook handler (non-static hook)
6) Fix spacing between the functions
7) Move below the `onRegistration()` callback below

I can help with some of this if needed also

Reply:
Should be done